letsencrypt/cli.py

```diff
@@ -723,10 +730,18 @@ def create_parser(plugins, args):
     return helpful.parser, helpful.args
 
 # For now unfortunately this constant just needs to match the code below;
-# there isn't an elegant way to autogenerate it in time.
-VERBS = ["run", "auth", "install", "revoke", "rollback", "config_changes",
-         "plugins", "--help"]
-
+# there isn't an elegant way to autogenerate it in time. pylint: disable=bad-whitespace
```

Comment:
rather than disabling the `bad-whitespace` (which in fact appies to the entire file, not only `VERBS`!), just fix the formatting not to indent; when we add new verb would you realign everything?